packages/validators/index.d.ts

```diff
@@ -83,18 +83,29 @@ export interface MessageProps {
   $propertyPath: string,
 }
 
-export type ValidatorWrapper = (...args: unknown[]) => ValidationRuleWithParams
+export type ValidatorWrapper = 
```

Comment:
I tried to look for a more generic solution, but in the end this was the best I could come up with